components/scream/src/dynamics/homme/interface/homme_grid_mod.F90

```diff
@@ -118,8 +119,8 @@ subroutine get_dyn_grid_data_f90 (gids_ptr, elgpgp_ptr, lat_ptr, lon_ptr) bind(c
           elgpgp(3,idof) = ip-1
 
           gids(idof) = INT(elem(ie)%gdofP(ip,jp),kind=c_int)
-          lat(idof)  = elem(ie)%spherep(ip,jp)%lat
-          lon(idof)  = elem(ie)%spherep(ip,jp)%lon
+          lat(idof)  = elem(ie)%spherep(ip,jp)%lat * 180.0/pi
```

Comment:
Did you intend to bring the radians->degrees change in this PR? Or was it by accident?

Reply:
I did, I made sure to explain why in the description of the PR.  I can change the title of the PR though so it doesn't get lost in the history where this change was made.

Reply:
No, I should just learn to fully read PRs description before starting the review... :)